### DISCUSSION TITLE
How to input video and get an output shot in video format

### DISCUSSION THREAD

**Discussion by aldnoahh** [li-plus/DSNet#7] [Q&A]
After trying to implement this repo for producing custom results in video formats, I have been unable to do so. If anyone has sucessfully done these, please guide me.

**Comment by atharvjairath** [comment]
> After trying to implement this repo for producing custom results in video formats, I have been unable to do so. If anyone has sucessfully done these, please guide me.

Same for me!

**Reply by wgqhandsome** [reply]
Hello, are you successful now?

**Comment by li-plus** [comment]
I have provided scripts for training & validation & inference on custom videos in the latest commit https://github.com/li-plus/DSNet/commit/1804176e2e8b57846beb063667448982273fca89. Please follow the instructions in `README.md` and check out the sample videos in `custom_data` folder. Hope it helps you. Further questions are welcome.

**Reply by atharvjairath** [reply]
@li-plus Thanks for that, It will make the task easier :)
